### DISCUSSION TITLE
clicking i3bar with `workspace_auto_back_and_forth` stopped working

### DISCUSSION THREAD

**Discussion by gbourant** [i3/i3#5751] [Q&A] [answered]
I have used for several years the following `workspace_auto_back_and_forth yes` and it was working great.

When you pressed `mod+$workspaceId` it switched to the workspace (still works). But also when you clicked at a workspace in i3bar (this stopped working).
 
![image](https://github.com/i3/i3/assets/9791051/69a2f74d-e615-473b-9a43-0a5d6cef6437)

Am i missing something? Has do to with i3wm (running atm v: 4.23-1)? Has to do with my linux installation?

Thanks in advance

**Comment by orestisfl** [comment, marked as answer]
This is probably a regression with 4.23: https://github.com/i3/i3/issues/5744

I'll try to fix when I am back at my computer.
